src/main/java/frc/robot/subsystems/lifter/factory/LifterFactory.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.lifter.factory;
+
+import frc.robot.Robot;
+import frc.robot.subsystems.lifter.LifterStuff;
+
+public class LifterFactory {
+
+	public static LifterStuff create(String logPath) {
+		return switch (Robot.ROBOT_TYPE) {
+			case REAL -> LifterRealConstants.generateLifterStuff(logPath);
+			case SIMULATION -> null;
```

Comment:
Why no simulation?

Reply:
cuz no simulation motor, we run only on robot.